doc/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -337,6 +337,12 @@ pimcore:
 - [Tool\Console] Methods `execInBackgroundUnix` & `execInBackgroundWindows` visibility changed from `protected` to `private` and for `getSystemEnvironment` from `public` to `private`.
 - [Translations] Added Symfony's html sanitizer to `\Pimcore\Model\Translation\Dao::save` method.
 - [Editable] Removed the `attributes` field from the link editable.
+- [Migrations] Pimcore does not run migrations after `composer` update automatically anymore. If you'd like to run all available migrations including bundles and your app-specific migrations, just include the following part in your `composer.json`.
+```json
+"post-update-cmd": [
+    "./bin/console doctrine:migrations:migrate"
+]
+```
```

Comment:
Is it recommended that all projects add this? If not, which projects should? When should it be added and when should it not be?

Reply:
Unfortunately a recommendation is not so easy. It really depends on the whole deployment process.
E.g. if you build a docker image, you basically do not want the migrations to be executed. 

The most important part is that you should make sure that the migrations will get executed because of the database changes that are included. How you do it is up to you. If your process involves a composer update and you are sure you want to execute the migrations then you can add it.
If your deployment process only used e.g. composer install then you need to add another line for the migrations to get executed.

Reply:
Thanks! I think that explanation would be great if it was included in the documentation.

Reply:
@NiklasBr I updated the it to include the explanation in general. Hopefully this helps!